### PR TITLE
fix(toolbar-save): 修复保存时红色 icon 相对位置丢失的 bug

### PR DESCRIPTION
**Pull request by chilingling**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

image

Issue Number: N/A

### What is the new behavior?
image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/toolbars/save/src/Main.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
     :content="isLoading ? '保存中' : '保存'"
   >
     <template #reference>
-      <span :class="[isLoading ? 'saving' : 'icon']" @click="openApi">
+      <span :id="`${isLoading ? 'saving' : ''}`" class="icon" @click="openApi">
```
Comment (ghost) [inline packages/toolbars/save/src/Main.vue:10]: 是不是可以就用动态class来设置样式，['icon', isLoading ? 'saving' : '' ]。

Comment (chilingling) [reply packages/toolbars/save/src/Main.vue:10]: 动态 class 优先级不够高，没法覆盖 `icon` 的样式

Comment (ghost) [reply packages/toolbars/save/src/Main.vue:10]: ok

**Review by ghost — APPROVED** [review]

File: packages/toolbars/save/src/Main.vue (hunk 1 above)
Comment (gene9831) [reply packages/toolbars/save/src/Main.vue:10]: 是否能用更具体的选择器来提高优先级，比如 `span.saving`

Comment (chilingling) [reply packages/toolbars/save/src/Main.vue:10]: @gene9831 `span.saving` 优先级还是不够高，需要嵌套多层，比如 `.xxxx-class{  div { span.saving } }` ，这样反而不利于后续的阅读和维护，所以选择用了 id

**Pull request merged**